test/records/test_models.py

```diff
@@ -65,98 +219,44 @@ def test_empty_for_optional_attributes(self):
         self.assertEqual(self.record.is_tna, False)
         self.assertEqual(self.record.is_digitised, False)
 
-    def test_iaid(self):
-
-        self.record = Record(self.source)
-
-        # patch raw data
-        self.record._raw["@template"]["details"]["iaid"] = "C123456"
-
+    def test_property_iaid(self):
+        response = APIResponse(self.source)
+        self.record = response.record
         self.assertEqual(self.record.iaid, "C123456")
 
-    def test_iaid_other_places(self):
-        self.record = Record(self.source)
-        # patch raw
-        self.record._raw["@template"]["details"]["@previous"] = {
-            "identifier": [
-                {
-                    "iaid": "C10298",
-                },
-            ],
-        }
-        self.record._raw["@template"]["details"]["@next"] = {
-            "identifier": [
-                {
-                    "iaid": "C10296",
-                },
-            ],
-        }
-        self.record._raw["@template"]["details"]["parent"] = {
-            "identifier": [
-                {
-                    "iaid": "C199",
-                },
-            ],
-        }
-
+    def test_property_previous(self):
```

Comment:
```suggestion
    def test_previous(self):
```
same comment as naming test with property. And follows for all names in this test file.